genesyscloud/resource_genesyscloud_routing_queue_test.go

```diff
@@ -56,15 +57,15 @@ func TestAccResourceRoutingQueueBasic(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				// Create
-				Config: GenerateUserWithCustomAttrs(testUserResource, testUserEmail, testUserName) + GenerateRoutingSkillResource(queueSkillResource, queueSkillName) +
+				Config: generateUserWithCustomAttrs(testUserResource, testUserEmail, testUserName) + GenerateRoutingSkillResource(queueSkillResource, queueSkillName) +
 					generateGroupResource(
 						bullseyeMemberGroupName,
-						"MySeries6Group",
+						"MySeries6Groupv2",
```

Comment:
Changed this name because a crash caused this resource to hand around in the test environment